lochness/transfer/__init__.py

```diff
@@ -317,6 +317,105 @@ def lochness_to_lochness_transfer_s3(Lochness, general_only: bool = True):
     logger.debug('aws rsync completed')
 
 
+def create_s3_transfer_table(Lochness, rewrite=False) -> None:
+    '''Extract s3 transfer information from the lochness log file
+
+    Key arguments:
+        Lochness: lochness object
+        rewrite: rewrite s3_log.csv if True, bool
+
+    Uses timestamp in the Lochness logfile to saves a s3_log.csv file under
+    PHOENIX root. This csv file has columns such as
+        - timestamp
+        - source
+        - destination
+        - filename
+        - protected
+        - study
+        - processed
+        - subject
+        - datatypes
```

Comment:
Comparing with Slack thread, I see only more column `ctime`. Is there any else?

Reply:
Only `ctime`. Thanks!